openpower/configs/linux/skiroot_defconfig

```diff
@@ -11,6 +11,7 @@ CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=20
 CONFIG_BLK_DEV_INITRD=y
+CONFIG_INITRAMFS_SOURCE="${BR_BINARIES_DIR}/rootfs.cpio"
```

Comment:
Note to self: This shouldn't be here.. it should be added by buildroot automatically, but was probably added by a `./op-build linux-update-config`